#! /bin/sh /usr/share/dpatch/dpatch-run
## remove-excessive-junk.dpatch by  <apollock@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Removes some unnecessary noise from the logs when logging errors

@DPATCH@
diff -urNad dhcp3-3.0.2/omapip/errwarn.c /tmp/dpep.f40ek4/dhcp3-3.0.2/omapip/errwarn.c
--- dhcp3-3.0.2/omapip/errwarn.c	2004-10-01 06:38:32.000000000 +1000
+++ /tmp/dpep.f40ek4/dhcp3-3.0.2/omapip/errwarn.c	2005-06-15 12:47:51.798490662 +1000
@@ -76,7 +76,7 @@
 	  write (STDERR_FILENO, "\n", 1);
   }
 
-#if !defined (NOMINUM)
+#if 0
   log_error ("%s", "");
   log_error ("If you did not get this software from ftp.isc.org, please");
   log_error ("get the latest from ftp.isc.org and install that before");
